lib/CustomFields/components/CustomFieldsForm/components/DeleteModal/DeleteModal.js

```diff
@@ -0,0 +1,116 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Modal,
+  ModalFooter,
+  Button,
+  Icon,
+  Layout,
+} from '@folio/stripes-components';
+
+import { FormattedMessage } from 'react-intl';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+const propTypes = {
+  fetchUsageStatistics: PropTypes.func.isRequired,
+  fieldsToDelete: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    index: PropTypes.number.isRequired,
+    name: PropTypes.string.isRequired,
+  })).isRequired,
+  onCancel: PropTypes.func.isRequired,
+  onConfirm: PropTypes.func.isRequired,
+  submitting: PropTypes.bool.isRequired,
+};
+
+const DeleteModal = ({
+  onConfirm,
+  onCancel,
+  fetchUsageStatistics,
+  fieldsToDelete,
+  submitting,
+}) => {
+  const [usageNumber, setUsageNumber] = useState(null);
+  const [usageNumberLoaded, setUsageNumberLoaded] = useState(false);
+
+  useEffect(() => {
+    const fetchStatistics = async () => {
+      const promises = fieldsToDelete.map(cf => {
```

Comment:
`cf` is not quite a readable name, please name it in a more readable way like `customField` as understant you meant. The same [here](https://github.com/folio-org/stripes-smart-components/pull/703/files#diff-fbb0b871db96bba556083674d43d8318R60), [here](https://github.com/folio-org/stripes-smart-components/pull/703/files#diff-47bbc820a330f2283c6d6ead7954b1e0R54), [here](https://github.com/folio-org/stripes-smart-components/pull/703/files#diff-7662134690a4a418782d3d9eb9a70961R68). Please check for other places where you better know it also exists.
```suggestion
      const promises = fieldsToDelete.map(cf => fetchUsageStatistics(cf.id));
```